src/EnergyPlus/DataDaylighting.hh

```diff
@@ -200,58 +202,48 @@ namespace DataDaylighting {
         Array4D<Real64> DaylSourceFacSunDisk;
         Array4D<Real64> DaylBackFacSun;
         Array4D<Real64> DaylBackFacSunDisk;
-        // Time exceeding maximum allowable discomfort glare index at reference points (hours)
-        Array1D<Real64> TimeExceedingGlareIndexSPAtRefPt;
         // Time exceeding daylight illuminance setpoint at reference points (hours)
         Array1D<Real64> TimeExceedingDaylightIlluminanceSPAtRefPt;
-        // True if at least one adjacent zone, sharing one or more interior windows, has daylighting control
-        bool AdjZoneHasDayltgCtrl;
-        int MapCount;          // Number of maps assigned to Zone
-        Array1D_int ZoneToMap; // Pointers to maps allocated to Zone
+        std::vector<std::vector<int>> ShadeDeployOrderExtWins; // describes how the fenestration surfaces should deploy the shades.
+        // It is a list of lists. Each sublist is a group of fenestration surfaces that should be deployed together. Many times the
+        // sublists a just a single index to a fenestration surface if they are deployed one at a time.
+        Array1D_int MapShdOrdToLoopNum; // list that maps back the original loop order when using ShadeDeployOrderExtWins for shade deployment
+        // Time exceeding maximum allowable discomfort glare index at reference points (hours)
+    };
 
-        // Default Constructor
-        ZoneDaylightCalc()
-            : DaylightMethod(iDaylightingMethod::NoDaylighting), AvailSchedNum(0), TotalDaylRefPoints(0), LightControlType(iLtgCtrlType::Continuous),
-              glareRefPtNumber(0), ViewAzimuthForGlare(0.0), MaxGlareallowed(0), MinPowerFraction(0.0), MinLightFraction(0.0), LightControlSteps(0),
-              LightControlProbability(0.0), TotalExtWindows(0), DElightGriddingResolution(0.0), ZonePowerReductionFactor(1.0), NumOfIntWinAdjZones(0),
-              NumOfIntWinAdjZoneExtWins(0), NumOfDayltgExtWins(0), MinIntWinSolidAng(0.0), InterReflIllFrIntWins(0.0), AdjZoneHasDayltgCtrl(false),
-              MapCount(0)
-        {
-        }
+    struct ZoneDaylightCalc
+    {
+        Real64 zoneAvgIllumSum = 0.0; // For VisualResilienceSummary reported average illuminance
+        int totRefPts = 0.0;          // For VisualResilienceSummary total number of rereference points
```

Comment:
I do like this reorganization but can't see from these changes why this is necessary. It would be nice for the team as a whole to understand why this was a good idea.